policyengine_us/variables/gov/states/ms/tax/income/adjustments/ms_retirement_income_exemption.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_us.model_api import *
+
+
+class ms_retirement_income_exemption(Variable):
+    value_type = float
+    entity = Person
+    label = "Mississippi retirement income exemption"
+    unit = USD
+    definition_period = YEAR
+    reference = [
+        "https://law.justia.com/codes/mississippi/title-27/chapter-7/article-1/section-27-7-15/",  # (4)(k)
+        "https://www.dor.ms.gov/sites/default/files/Forms/Individual/80100221.pdf#page=11",  # Line 46
+        "https://www.dor.ms.gov/sites/default/files/Forms/Individual/2024/80105248.pdf",
+    ]
+    defined_for = StateCode.MS
+
+    def formula(person, period, parameters):
+        excluded_income = add(
+            person,
+            period,
+            [
+                "taxable_social_security",
+                "taxable_pension_income",
```

Comment:
I don't think this should be included. (4)(k) only exempts taxable Social Security benefits and public pensions. Line 46 requires entering taxable pensions.

Reply:
That's one example but I don't see it as comprehensive. I'll ask on the other issue.

Reply:
keeping as-is as (4)(l) suggests full pension exemption, which taxact applies